src/content/pages/capi.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: C API Summit
+subtitle: A summit to bring together users (and interested people) of the C API that Python provide, to discuss the state of this ecosystem, existing challenges, and ongoing work.
+---
+
+# C API Summit
+
+* **When**: Monday, July 8th.
+* **Where**: Prague Congress Centre (PCC), Room Club D
+* **Who can join**: Anyone with a valid in-person EuroPython 2024 ticket - Conference, Tutorial or Combined (see [ticket types](https://ep2024.europython.eu/tickets#ticket-types) for details).
+
+The C API has many different stakeholders, including but not limited to
+CPython core developers, maintainers of Python extensions, code generators
+(Cython, pybindy11, etc) and alternative API design (HPy)
+
+The main goal of this summit is to make sure that people from all these
+different projects can meet and discuss about the state of the C API, existing
+challenges and ongoing work.
+
+## Agenda
+
+* 9:00: Meet and greet - nconference-y post-it based organisation.
+* 9:30: Presentations (30mins each)
+* 11:00: Coffee
+* 11:20: Presentations
```

Comment:
```suggestion
* 11:15: Presentations
```
the official coffee break according to the schedule is until 11:15 . - But it's up to the organisers of the summit if they want to align or use a different timeline of course.

Reply:
thank you for pointing it out. I think it's a good idea to align to the official schedule, so I'm +1 to accept all the proposed changes (but I can't because I'm not the author of the PR).
/cc @cmaureir